api/service/projects_service_test.go

```diff
@@ -83,6 +83,7 @@ func TestProjectsService_CreateProject(t *testing.T) {
 
 			projectResource := fmt.Sprintf(ProjectResources, tt.arg.Id)
 			projectSubResource := fmt.Sprintf(ProjectSubResources, tt.arg.Id)
+			projectNameResource := fmt.Sprintf(ProjectResources, tt.arg.Name)
```

Comment:
Shouldn't you use `ProjectSubResources` ? With current one it will be something like `resources:mlp:projects:my_project` which doesn't include any resource under `my_project`?